### PR TITLE
SSR-2360: Custom css button and text

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

File: demos/esm-dev-app/src/index.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+  visible: true,
+  text: "text",
+  disabled: false
+});
+root.appendChild(button);
+
+const text = new Text({
+  value: "Orange",
+  className: "text-class",
+  id: "aaaaaa",
```
Comment (nmanhit) [inline demos/esm-dev-app/src/index.js:16]: text-id

File: demos/esm-dev-app/src/index.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  textalign: "righttt",
```
Comment (nmanhit) [inline demos/esm-dev-app/src/index.js:17]: textalign: "right"

File: demos/esm-dev-app/src/index.js [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+  placeholder: "hogehoge1",
```
Comment (nmanhit) [inline demos/esm-dev-app/src/index.js:18]: placeholder: "placeholder"

File: demos/esm-dev-app/dist/styles.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    --kuc-button-width: 100px;
+    --kuc-button-height: 100px;
+    --kuc-button-background-color: rgb(122, 22, 22);
+    --kuc-button-background-hover: yellow;
+    --kuc-button-background-active: blue;
+    --kuc-button-background-focus: green;
+    --kuc-button-text-color: white;
+    --kuc-button-font-size: 18px;
+}
+.text-class {
```
Comment (nmanhit) [inline demos/esm-dev-app/dist/styles.css:11]: Where is it used?

File: src/button/style.ts [hunk 5]
```diff
@@ -23,55 +23,80 @@ export const BUTTON_CSS = `
...
+  }
+  .kuc-button__button--alert:active {
+    background-color: var(--kuc-button-background-active, #bf2718);
+  }
   .kuc-button__button:disabled {
+    background-color: var(--kuc-button-background-color, #d4d7d7);
     border: 1px solid #e3e7e8;
-    background-color: #d4d7d7;
     box-shadow: none;
     color: #888888;
```
Comment (nmanhit) [inline src/button/style.ts:101]: Do we need to update
color: var(--kuc-button-text-color, #888888);

Comment (voduchau) [reply src/button/style.ts:101]: Yes, I think we should update it too

**Pull request merged**